MoreLinq/OrderedMerge.cs

```diff
@@ -70,7 +70,7 @@ static partial class MoreEnumerable
             IEnumerable<T> second,
             IComparer<T>? comparer)
         {
-            return OrderedMerge(first, second, e => e, f => f, s => s, (a, _) => a, comparer);
+            return OrderedMerge(first, second, Identity<T>, Identity<T>, Identity<T>, (a, _) => a, comparer);
```

Comment:
Type argument is redundant:

```suggestion
            return OrderedMerge(first, second, Identity, Identity, Identity, (a, _) => a, comparer);
```